Error starting 24-hour cleaning: (raised as a result of Query-invoked autoflush; consider using a session.no_autoflush block if this flush is occurring prematurely) (psycopg2.errors.UndefinedColumn) column "timer_active" of relation "cleaning_process" does not exist LINE 1: ...e_name, operator_name, status, notes, created_at, timer_acti... ^ [SQL: INSERT INTO cleaning_process (order_id, cleaning_bin_id, process_type, duration_hours, start_time, end_time, actual_end_time, start_moisture, end_moisture, target_moisture, water_added_liters, waste_collected_kg, machine_name, operator_name, status, notes, created_at, timer_active, countdown_start, countdown_end, buffer_time_minutes, buffer_start_time, buffer_end_time, is_locked, reminder_sent_5min, reminder_sent_10min, reminder_sent_30min, start_parameters_captured, completion_parameters_captured, moisture_before, moisture_after, waste_material_kg, water_used_liters, machine_efficiency, post_process_notes, completed_by, completion_time) VALUES (%(order_id)s, %(cleaning_bin_id)s, %(process_type)s, %(duration_hours)s, %(start_time)s, %(end_time)s, %(actual_end_time)s, %(start_moisture)s, %(end_moisture)s, %(target_moisture)s, %(water_added_liters)s, %(waste_collected_kg)s, %(machine_name)s, %(operator_name)s, %(status)s, %(notes)s, %(created_at)s, %(timer_active)s, %(countdown_start)s, %(countdown_end)s, %(buffer_time_minutes)s, %(buffer_start_time)s, %(buffer_end_time)s, %(is_locked)s, %(reminder_sent_5min)s, %(reminder_sent_10min)s, %(reminder_sent_30min)s, %(start_parameters_captured)s, %(completion_parameters_captured)s, %(moisture_before)s, %(moisture_after)s, %(waste_material_kg)s, %(water_used_liters)s, %(machine_efficiency)s, %(post_process_notes)s, %(completed_by)s, %(completion_time)s) RETURNING cleaning_process.id] [parameters: {'order_id': 3, 'cleaning_bin_id': 1, 'process_type': '24_hour', 'duration_hours': 0.1, 'start_time': datetime.datetime(2025, 9, 11, 13, 7, 32, 964280), 'end_time': datetime.datetime(2025, 9, 11, 13, 13, 32, 964280), 'actual_end_time': None, 'start_moisture': None, 'end_moisture': None, 'target_moisture': None, 'water_added_liters': 0.0, 'waste_collected_kg': 0.0, 'machine_name': 'kjlkj', 'operator_name': 'jk', 'status': 'running', 'notes': None, 'created_at': datetime.datetime(2025, 9, 11, 13, 7, 32, 971648), 'timer_active': False, 'countdown_start': None, 'countdown_end': None, 'buffer_time_minutes': 0.0, 'buffer_start_time': None, 'buffer_end_time': None, 'is_locked': True, 'reminder_sent_5min': False, 'reminder_sent_10min': False, 'reminder_sent_30min': False, 'start_parameters_captured': False, 'completion_parameters_captured': False, 'moisture_before': None, 'moisture_after': None, 'waste_material_kg': None, 'water_used_liters': None, 'machine_efficiency': None, 'post_process_notes': None, 'completed_by': None, 'completion_time': None}] (Background on this error at: https://sqlalche.me/e/20/f405)